blaze-client/src/main/scala/org/http4s/client/blaze/BasicManager.scala

```diff
@@ -0,0 +1,23 @@
+package org.http4s
+package client
+package blaze
+
+import scalaz.concurrent.Task
+
+
+/** Create a basic [[ConnectionManager]] that creates new connections on each request */
+object BasicManager {
+  def apply(builder: ConnectionBuilder): ConnectionManager =
```

Comment:
Maybe `ConnectionManager.basic` and `ConnectionManager.pooled`?  I wouldn't necessarily know where to look for the scattered connection managers.